features/admin.server-configurations.v1/forms/sift-connector-form/sift-connector-form.tsx

```diff
@@ -0,0 +1,219 @@
+                                }
+                                placeholder={ t("governanceConnectors:connectorCategories" +
+                                    ".loginAttemptsSecurity.connectors.siftConnector" +
+                                    ".properties.siftConnectorApiKey.placeholder")
+                                }
+                                component={ TextFieldAdapter }
+                                autoComplete="new-password"
+                                InputProps={ {
+                                    endAdornment: renderInputAdornment()
+                                } }
+                                initialValue={ initialValues?.properties[0]?.value ?? "" }
+                                onChange={ (e: React.ChangeEvent<HTMLInputElement>) => {
+                                    const value: string = e.target.value;
+
+                                    setApiKey(value);
+                                    form.change(GovernanceConnectorUtils
+                                        .encodeConnectorPropertyName(ServerConfigurationsConstants
+                                            .SIFT_CONNECTOR_API_KEY_PROPERTY), value
+                                    );
+                                } }
+                                value={ apiKey }
+                            />
+                            {
+                                (apiKey && apiKey !== "") && (
+                                    <IconButton
+                                        className="delete-button"
+                                        onClick={ () => {
+                                            setApiKey("");
+                                            form.change(
+                                                GovernanceConnectorUtils
+                                                    .encodeConnectorPropertyName(ServerConfigurationsConstants
+                                                        .SIFT_CONNECTOR_API_KEY_PROPERTY), ""
+                                            );
+                                        } }
+                                        data-componentid={ `${ componentId }-configuration-form-api-key-delete-button` }
+                                    >
+                                        <TrashIcon />
+                                    </IconButton>
+                                )
+                            }
+                        </Stack>
+                        <PrimaryButton
+                            className={ "mr-3 mt-3" }
+                            size="small"
+                            disabled={ isSubmitting || readOnly }
+                            loading={ isSubmitting }
+                            data-componentid={
+                                `${ componentId }-configuration-form-submit-button`
+                            }
+                        >
+                            { t("common:update").toString() }
```

Comment:
Hope we don't need `toString` here.

```suggestion
                            { t("common:update") }
```